### ISSUE TITLE
Type error in notification config

### ISSUE THREAD

**Issue by hasib-devs** [verful/adonis-notifications#8]
```typescript
const NotificationConfig: NotificationConfig = {
  channel: 'database',
  channels: {
    database: {
      driver: 'database',
    },
    mail: {
      driver: 'mail',
      mailer: 'smtp' // Error: Type '{ driver: "mail"; mailer: string; }' is not assignable to type 'MailChannelConfig'.
    },
  },
  notificationsTable: 'notifications' // Error notificationsTable: string; }' is not assignable to type 'NotificationConfig'.
}
```

**Comment by oluudeh** [comment]
Removing the `mailer` and `notificationsTable` properties seems to work for me. Like so

```
const NotificationConfig: NotificationConfig = {
  channel: 'database',
  channels: {
    /*
    |--------------------------------------------------------------------------
    | Database channel
    |--------------------------------------------------------------------------
    |
    | Use this channel to store notifications in the database.
    |
    */
    database: {
      driver: 'database',
    },
    /*
    |--------------------------------------------------------------------------
    | Mail channel
    |--------------------------------------------------------------------------
    |
    | Use this channel to send notifications via email.
    |
    */
    mail: {
      driver: 'mail',
    },
  },
}

export default NotificationConfig
```

**Comment by maukoese** [comment]
> Removing the `mailer` and `notificationsTable` properties seems to work for me. Like so
> 
> ```
> const NotificationConfig: NotificationConfig = {
>   channel: 'database',
>   channels: {
>     /*
>     |--------------------------------------------------------------------------
>     | Database channel
>     |--------------------------------------------------------------------------
>     |
>     | Use this channel to store notifications in the database.
>     |
>     */
>     database: {
>       driver: 'database',
>     },
>     /*
>     |--------------------------------------------------------------------------
>     | Mail channel
>     |--------------------------------------------------------------------------
>     |
>     | Use this channel to send notifications via email.
>     |
>     */
>     mail: {
>       driver: 'mail',
>     },
>   },
> }
> 
> export default NotificationConfig
> ```

Let me try this.

**Comment by arthur-er** [comment]
Fixed in 2.1.0, please re-run `node ace configure` and the config should be replaced with a working version.
